stockpile/roles/foreman/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+  # Capture foreman related data
+  - name: Get foreman min instances
+    shell:
+       grep "PassengerMinInstances" /etc/httpd/conf.d/05-foreman-ssl.conf | cut -d ' ' -f 4
+    register: foreman_min_instances
+    ignore_errors: true
+  - name: Get foreman start timeout
+    shell:
+       grep "PassengerStartTimeout" /etc/httpd/conf.d/05-foreman-ssl.conf | cut -d ' ' -f 4
+    register: foreman_start_timeout
+    ignore_errors: true
+  - name: Capture foreman keepalive settings
+    shell:
+       grep "KeepAlive" /etc/httpd/conf.d/05-foreman-ssl.conf | cut -d ' ' -f 4
+    register: foreman_keepalive
+    ignore_errors: true
+  - shell:
+       grep "KeepAliveTimeout" /etc/httpd/conf.d/05-foreman-ssl.conf | cut -d ' ' -f 4
+    register: foreman_keepalive_timeout
+    ignore_errors: true
+  - shell:
+       grep "MaxKeepAliveRequests" /etc/httpd/conf.d/05-foreman-ssl.conf | cut -d ' ' -f 4
+    register: max_keepalive_requests
```

Comment:
max_keepalive_requests is too generic, same goes for executors_count and executors_memory_limit